lib/lightning_web/live/project_live/settings.html.heex

```diff
@@ -114,6 +116,31 @@
                 </small>
               </div>
             </div>
+            <div class="hidden sm:block" aria-hidden="true">
+              <div class="py-2"></div>
+            </div>
+            <div class="grid grid-cols md:col-span-2">
+              <div class="flex flex-row">
+                <div>
+                  <.label>Disable parallel run execution?</.label>
+                </div>
+                <div class="ml-2 mt-1">
+                  <.input
+                    type="checkbox"
+                    field={f[:concurrency]}
+                    disabled={!@can_edit_project}
+                    label=""
+                    checked_value="1"
+                    unchecked_value={nil}
+                  />
+                </div>
+              </div>
+              <div>
+                <small class="block my-1 text-xs text-gray-600">
+                  Additionally you can setup all runs from this project, accross multiple workflows, to not execute in parallel.
```

Comment:
Could we replace the word "Additionally" by "Note that", for more clarity on the note ?